flow/src/org/labkey/flow/controllers/editscript/editCompensationCalculation.jsp

```diff
@@ -76,7 +75,7 @@ var SS = []; // SUBSETS
             if (!hashToIndexMap.containsKey(subsetHash))
             {
                 hashToIndexMap.put(subsetHash, index);
-                index = index.intValue() + 1;
+                index = (int)index + 1;

```

Comment:
This cast looks redundant

Reply:
I noticed that as well... already fixed it in my Daily change list. IntelliJ made the Integer → int suggestion, odd that it would add the cast.